eslint.config.mjs

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+import shopifyEslintPlugin from '@shopify/eslint-plugin';
+
+export default [
+  {
+    // global ignores need to be in their own block otherwise they don't seem to work
+    ignores: [
+      '.docker/**',
+      '.github/**',
+      '.vscode/**',
+      'coverage/**',
+      'dist/**',
+      'node_modules/**',
+      '**/*.config.{mjs,ts}'
+    ],
+  },
+  ...shopifyEslintPlugin.configs.typescript,
+  ...shopifyEslintPlugin.configs.prettier,
+  {
+    files: ['**/entities/*.ts'],
+    rules: {
+      'import/no-cycle': 'off',
+    }
+  },
+  {
+    rules: {
+      'line-comment-position': 'off',
+      'no-process-env': 'warn',
+      'no-warning-comments': 'warn',
+      '@typescript-eslint/member-ordering': 'warn',
+      '@typescript-eslint/naming-convention': [
+        'error',
+        {
+          selector: 'default',
+          format: ['camelCase', 'PascalCase', 'UPPER_CASE', 'snake_case'],
```

Comment:
default seems to be `['camelCase', 'PascalCase', 'UPPER_CASE']` which gives us a lot of errors due to the DTOs.